config/locales/en.yml

```diff
@@ -135,9 +135,6 @@ en:
     open_state_link_text: "Request PE courses for %{period}"
     closed_state_link_text: "Request PE courses for %{period}"
     confirmed_state_link_text: "View PE courses for %{period}"
-    open_state_copy: "Only accredited bodies are able to see this section. Use it to request fee-funded PE courses (ones without a salary) for the next recruitment cycle."
-    closed_state_copy: "Only accredited bodies are able to see this section. Use it to request fee-funded PE courses (ones without a salary) for the next recruitment cycle."
-    confirmed_state_copy: "Use it to view the outcome of requests for fee-funded PE courses (ones without a salary) for the next recruitment cycle."
```

Comment:
Just want to confirm that we want to remove these variations (the prototype doesn’t cover these states, I don’t think). Looks like these are replaced by the single line:

> If you accredit fee-funded PE courses for any training providers, you must request these courses on their behalf.

Is this correct @B-Bonner (looks like it might be)?

Reply:
I only updated the variation that appeared on the prototype page. I left the others as I didn't understand the context for them (and couldn't get an answer). Does that answer your question @paulrobertlloyd ?

Reply:
But the change I made to that instance is correct, yes.